Uno.Gallery/Uno.Gallery.UITest/TestBase.cs

```diff
@@ -25,9 +26,11 @@ static TestBase()
             AppInitializer.TestEnvironment.AndroidAppName = Constants.AndroidAppName;
             AppInitializer.TestEnvironment.iOSDeviceNameOrId = Constants.iOSDeviceNameOrId;
             AppInitializer.TestEnvironment.CurrentPlatform = Constants.CurrentPlatform;
+			AppInitializer.TestEnvironment.ChromeDriverPath = @"C:\Program Files\Google\Chrome\chromedriver-win64";
```

Comment:
move this to the `Constants`

Reply:
This should only be needed for your case I believe, we shouldn't include it as part of the source

Reply:
> This should only be needed for your case I believe, we shouldn't include it as part of the source
Should i remove the ChromeDriverPath ? @kazo0